src/client/pages/create.js

```diff
@@ -225,11 +242,18 @@ class Create extends React.Component {
                 <td>
                   <a href={`${window.location.origin}/${this.state.gameID}/${i}/${this.state.secret[i]}`} target="_blank" rel="noopener noreferrer">{window.location.origin}/{this.state.gameID}/{i}/{this.state.secret[i]}</a>
                 </td>
+                <td>
+                  <Button onClick={() => this.copyToClipboard(`${window.location.origin}/${this.state.gameID}/${i}/${this.state.secret[i]}`)}>
+                    Copy
+                  </Button>
+                </td>
               </tr>
             )}
             </tbody>
           </Table>
           <hr />
+          <Button onClick={this.copyAllLinks} color="warning" block size="lg">Copy All</Button>
```

Comment:
Just a general comment:  Is there some way to write a test for this?

Reply:
Could you also implement the color effect for this button?

Reply:
In terms of testing, I can't work out an easy way to test the stuff I've added

Reply:
One way might be to make `copyToClipboard` a property in `<CopyButton />` (see "dependency injection").
Then, in a test, you could pass in a mock and see whether it is called with the correct arguments.